ConcernsCaseWork/Service.Redis/NtiWarningLetter/INtiWarningLetterCachedService.cs

```diff
@@ -12,5 +12,7 @@ public interface INtiWarningLetterCachedService
 		Task<ICollection<NtiWarningLetterDto>> GetNtiWarningLettersForCaseAsync(long caseUrn);
 		Task<NtiWarningLetterDto> GetNtiWarningLetterAsync(long ntiWarningLetterId);
 		Task<NtiWarningLetterDto> PatchNtiWarningLetterAsync(NtiWarningLetterDto ntiWarningLetter);
+		Task SaveNtiWarningLetterToCache(NtiWarningLetterDto ntiWarningLetter, string continuationId);
```

Comment:
This is the CachedService, so not sure we need to specify that we're saving to the cache in the method name?